app/src/main/java/org/mifos/mobile/ui/fragments/LoanAccountSummaryFragment.kt

```diff
@@ -110,73 +32,73 @@ class LoanAccountSummaryFragment : BaseFragment() {
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                               savedInstanceState: Bundle?): View? {
-        rootView = inflater.inflate(R.layout.fragment_loan_account_summary, container, false)
+        _binding = FragmentLoanAccountSummaryBinding.inflate(inflater, container, false)
         setToolbarTitle(getString(R.string.loan_summary))
-        ButterKnife.bind(this, rootView!!)
         showLoanAccountsDetail(loanWithAssociations)
-        return rootView
+        return binding.root
     }
 
     /**
      * Sets basic information about a Loan Account
      *
      * @param loanWithAssociations object containing details of each loan account,
      */
-    fun showLoanAccountsDetail(loanWithAssociations: LoanWithAssociations?) {
-        llLoanSummary?.visibility = View.VISIBLE
+    private fun showLoanAccountsDetail(loanWithAssociations: LoanWithAssociations?) {
+        binding.llLoanSummary.visibility = View.VISIBLE
```

Comment:
You could use Kotlin's `with` and avoid using the `binding` variable many times inside the function.

```
with(binding){
  llLoanSummary.visibility = View.VISIBLE
  tvLoanProductName.text = loanWithAssociations?.loanProductName
 ....
}
```

Reply:
Thanks @jawidMuhammadi, got it. Made the changes.

Reply:
@gururani-abhishek and @PratyushSingh07 whenever there is a code change request from someone on your PR, please don't mark it as resolved. Once it is done, ping the reviewer and let them mark it as resolved.

Reply:
alright, got it.